src/compose/ComposeBox.js

```diff
@@ -441,6 +546,68 @@ export default function ComposeBox(props: Props): Node {
     messageInputState,
   ]);
 
+  const doQuoteAndReply = useCallback(
```

Comment:
Code-ordering-wise, let's put this right after `insertAttachment`.
* They're very similar in the nature of how they're implemented, so it's helpful to have them next to each other for comparison.
* The functions above this point and below `insertAttachment` are largely UI handler callbacks, so they're naturally more closely tied to the JSX below than this is.